postgres-appliance/build_scripts/prepare.sh

```diff
@@ -30,26 +30,18 @@ fi
 # Cleanup all locales but en_US.UTF-8 and optionally specified in ADDITIONAL_LOCALES arg
 find /usr/share/i18n/charmaps/ -type f ! -name UTF-8.gz -delete
```

Comment:
These lines are already in the [locales.sh](https://github.com/zalando/spilo/pull/770/files#diff-0f1e719f5a23de80bcd623f061a08c2d7726e2976c880e6b752f4c3a4b220206R13-R14) and should be removed from here.